Formula/gobject-introspection.rb

```diff
@@ -14,10 +14,7 @@ class GobjectIntrospection < Formula
   depends_on "glib"
   depends_on "cairo"
   depends_on "libffi"
-  # System python in Mavericks or below has bug in distutils/sysconfig.py, which breaks the install.
-  #    Caught exception: <type 'exceptions.AttributeError'> AttributeError("'NoneType' object has no attribute 'get'",)
-  #    > /System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/distutils/sysconfig.py(171)customize_compiler()
-  depends_on "python" if MacOS.version <= :mavericks
+  depends_on "python" # never switch back to system python!
```

Comment:
No, this is my personal experience speaking 😄 

A lot of gnome projects are migrating to meson as build system, and I noticed that when these projects need to generate gobject-introspection bindings, this fails because the system python adds flags that trigger a universal build of these bindings, which screws with some glib macros.